MapboxNavigation/PollyVoiceController.swift

```diff
@@ -166,7 +172,7 @@ public class PollyVoiceController: RouteVoiceController {
         }
     }
     
-    func callSuperSpeak(_ text: String, error: String) {
+    func callSuperSpeak(_ text: String, error: Error) {
```

Comment:
`speakWithoutPolly(for:userDistance:)`

https://github.com/mapbox/mapbox-navigation-ios/blob/2097f6d5cb95cdfa154d5c56560a8c0ffa8e107e/MapboxNavigation/PollyVoiceController.swift#L120